### PR TITLE
[JetBrains] Update Platform Version from JetBrains Gateway Plugin (EAP)

### PR DESCRIPTION
**Pull request by akosyakov**
## Description
This PR updates the Platform Version from JetBrains Gateway Plugin (EAP) to the latest version.

## How to test

Merge if tests are green, if something breaks then add tests for regressions.

<details>
<summary>if you want to test manually for some reasons</summary>

1. Ensure you have the Gateway installed from [JetBrains Toolbox App](https://www.jetbrains.com/toolbox-app/) and have it up-to-date.
  - You should use Gateway version corresponding to plugin qualifier, i.e. for stable plugin test with released, for latest test with EAP.
  - It could be that a new Gateway is not published for the given SDK yet then wait for it to be published. You can check the build version in the About dialog.
2. Download the plugin build related to this branch in [Dev Versions](https://plugins.jetbrains.com/plugin/18438-gitpod-gateway/versions/dev), and [install it on the Gateway](https://www.jetbrains.com/help/idea/managing-plugins.html#install_plugin_from_disk).
  - You can also uninstall current Gitpod plugin, configure dev channel using https://plugins.jetbrains.com/plugins/list?channel=dev&pluginId=18438-gitpod-gateway and restart GW to test that the proper version is detected and installed.
3. Create a new workspace from the Gateway (it's ok to use the pre-selected IDE and Repository) and confirm if JetBrains Client can connect to it.
</details>

## Release Notes
```release-note
NONE
```

## Werft options:
- [x] /werft publish-to-jb-marketplace
- [x] /werft with-preview
- [x] /werft with-large-vm
- [x] /werft with-gce-vm
- [x] with-integration-tests=all
- [x] latest-ide-version=true
- [x] with-ws-manager-mk2

_This PR was created automatically with GitHub Actions using [this](https://github.com/gitpod-io/gitpod/blob/main/.github/workflows/jetbrains-update-plugin-platform-template.yml) template._

### CODE REVIEW

File: test/pkg/integration/apis.go [hunk 1]
```diff
@@ -491,12 +491,11 @@ func (c *ComponentAPI) CreateUser(username string, token string) (string, error)
 	}
 	if authId == "" {
 		authId = strconv.FormatInt(time.Now().UnixMilli(), 10)
-		_, err = db.Exec(`INSERT IGNORE INTO d_b_identity (authProviderId, authId, authName, userId, tokens) VALUES (?, ?, ?, ?, ?)`,
```
Comment (akosyakov) [inline test/pkg/integration/apis.go:494]: @kylos101 Could you approve please to unblock JB update? 

For context see https://gitpod.slack.com/archives/C032A46PWR0/p1685973787702279

Comment (kylos101) [reply test/pkg/integration/apis.go:494]: @akosyakov do jetbrains tests run with `with-integration-tests=all`? I ask to assert that workspace tests are still working, too.

Comment (akosyakov) [reply test/pkg/integration/apis.go:494]: Can be done with another PR? That we don't blcok JB upgade withing for build again?

Comment (akosyakov) [reply test/pkg/integration/apis.go:494]: nvm they are failing any more, let's rerun all of them

Comment (akosyakov) [reply test/pkg/integration/apis.go:494]: @kylos101 All JB tests are passing (@iQQBot can we get an approve on this PR from IDE team 🙏 ), I found following failed other tests not related to IDEs:
```
2023-06-05T19:22:15.0182695Z [test-server]         --- FAIL: TestStartWorkspace/CreateWorkspace/it_can_run_workspace_tasks (19.49s)
...
2023-06-05T20:36:03.0303764Z [test-workspace]             --- FAIL: TestGitHubContexts/context/should_run_context_tests/github.com/gitpod-io/gitpod-test-repo/tree/integration-test-1 (509.23s)
...
2023-06-05T20:43:33.9819932Z [test-workspace]         --- FAIL: TestRunDocker/docker/it_should_start_a_container (300.44s)
...
2023-06-05T20:50:55.9688839Z [test-workspace]             --- FAIL: TestGitActions/GitActions/it_can_run_git_actions/github.com/gitpod-io/gitpod-test-repo/tree/integration-test/commit_classic (150.57s
```

**Comment by kylos101** [comment]
@akosyakov this is deployed in gen99, we're shifting traffic now

**Pull request merged**